rdkit/VLib/Filter.py

```diff
@@ -3,10 +3,10 @@
 #  Copyright (C) 2003 Rational Discovery LLC
 #     All Rights Reserved
 #
-import sys
+
+import six
```

Comment:
the reasons for a bundled six module are mainly related to this small customization

https://github.com/rdkit/rdkit/blob/master/rdkit/six.py#L209

that was helpful to address some portability issues we faced while unpickling some data files used by the test code.

to a very much lesser extent, it also helps avoiding an additional external dependency